[STORM-2644] Show message when result not found on deep search page

Ethanlm opened the pull request:
Deep search shows a blank area if no results are found. see https://issues.apache.org/jira/browse/STORM-2644

![image](https://user-images.githubusercontent.com/14900612/28389389-82e92d84-6c9b-11e7-9594-8e9690e22724.png)

Show some message to avoid confusion about whether the searching is finished or not.

Ethanlm commented:
![image](https://user-images.githubusercontent.com/14900612/28389361-69561454-6c9b-11e7-8cd2-7f5aeafa373f.png)

knusbaum commented:
I would prefer *one* message if nothing is found. When there are hundreds of workers, this would make users scroll through many "Result not found" messages before getting to results that *are* returned.

Ethanlm commented:
Thanks. Will fix it

Ethanlm commented:
Addressed comment from @knusbaum . 
Tested some simple cases on firefox, chrome and safari. The following screenshots are from testing on firefox.
![onlyon6702](https://user-images.githubusercontent.com/14900612/28425598-654cf5d0-6d36-11e7-8f25-71449922f6b8.png)
![onlyon6703](https://user-images.githubusercontent.com/14900612/28425602-67de6dba-6d36-11e7-80e9-7980ba698f8d.png)
![notanywhere](https://user-images.githubusercontent.com/14900612/28425608-6a0f01da-6d36-11e7-8a19-007796b7d33a.png)
![onbothworkers](https://user-images.githubusercontent.com/14900612/28425614-6c1cefbe-6d36-11e7-8b06-36087f1f31d9.png)

HeartSaVioR commented:
+1
